reactor-core/src/main/java/reactor/core/publisher/FluxFilter.java

```diff
@@ -230,7 +245,10 @@ public boolean tryOnNext(T t) {
 				b = predicate.test(t);
 			}
 			catch (Throwable e) {
-				onError(Operators.onOperatorError(s, e, t, actual.currentContext()));
+				Throwable e_ = Operators.onNextFailure(t, e, actual.currentContext(), s);
```

Comment:
same thing that previous comment adapted for ConditionalSubscriber

Reply:
I did it that way because some operators slightly deviate from a "standard" actual.onError implementation. For instance, consider `FluxMap`:

```java
Throwable e_ = Operators.onNextError(t, e, actual.currentContext(), s);
				if (e_ != null) {
					done = true;
					actual.onError(e_);
					return true;
				}
				else {
					return false;
				}
```